mitodl/open-discussions
# LoginPopup for Follow button

#### Pre-Flight checklist


#### What are the relevant tickets?
Partially addresses #1051

#### What's this PR do?
- Adds the `LoginPopup` component to the follow button
- Creates a new class for `FollowButton` instead of creating it via a function within ExpandedPostDisplay.
- Adds `className` to the LoginPopup so its styling can be customized (necessary for the Follow button, to avoid it overflowing past the right edge of the browser window).

#### How should this be manually tested?
- When logged in, the Follow/Unfollow button should work as before on the post detail page.
- When not logged in, the Follow button should still display and clicking it should display the login popup.
